docs/book/src/reference/versions.md

```diff
@@ -77,20 +77,23 @@ The Kubeadm Bootstrap Provider generates kubeadm configuration using the API ver
+| -------------------------- | ------------------------------- |-------------------------------|---------------------------------|-------------------------------| ------------------------------- | ----------------------------- |
 | Kubernetes v1.16 + etcd/v3 | ✓                               | ✓                             |                                 |                               |                                 |                               |
 | Kubernetes v1.17 + etcd/v3 | ✓                               | ✓                             |                                 |                               |                                 |                               |
 | Kubernetes v1.18 + etcd/v3 | ✓                               | ✓                             |                                 | ✓                             |                                 | ✓                             |
 | Kubernetes v1.19 + etcd/v3 | ✓                               | ✓                             | ✓                               | ✓                             | ✓                               | ✓                             |
 | Kubernetes v1.20 + etcd/v3 | ✓                               | ✓                             | ✓                               | ✓                             | ✓                               | ✓                             |
 | Kubernetes v1.21 + etcd/v3 | ✓                               | ✓                             | ✓                               | ✓                             | ✓                               | ✓                             |
-| Kubernetes v1.22 + etcd/v3 |                                 | ✓                             | ✓                               | ✓                             | ✓                               | ✓                             |
-| Kubernetes v1.23 + etcd/v3 |                                 |                               | ✓                               | ✓                             | ✓                               | ✓                             |
+| Kubernetes v1.22 + etcd/v3 |                                 | ✓*                            | ✓                               | ✓                             | ✓                               | ✓                             |
+| Kubernetes v1.23 + etcd/v3 |                                 |                               | ✓*                              | ✓*                            | ✓                               | ✓                             |
 
 The Kubeadm Control Plane Provider talks to the API server and etcd members of every Workload Cluster whose control plane it owns. It uses the etcd v3 API.
 
 The Kubeadm Control Plane requires the Kubeadm Bootstrap Provider.
 
+\* Newer versions of CoreDNS may not be compatible as an upgrade target for clusters managed with Cluster API. Versions marked on the table are supported as an upgrade target only if CoreDNS is not upgraded to the latest version. 
```

Comment:
```suggestion
\* Newer versions of CoreDNS may not be compatible as an upgrade target for clusters managed with Cluster API. Versions marked on the table are supported as an upgrade target only if CoreDNS is not upgraded to the latest version supported by the respective Kubernetes version. 
```
Maybe a slight improvement, up to you.

Reply:
I think so - but I'll wait to see if this is overall clear enough. before making the change.

Reply:
Let's just add the following table
- CAPI v1alpha3 (v0.3): max CoreDNS version to be used for upgrades v1.8.4 
- CAPI v1alpha4 (v0.4): max CoreDNS version to be used for upgrades v1.8.4
- CAPI v1beta1 (v1.0): max CoreDNS version to be used for upgrades v1.8.5
- CAPI v1alpha4 (v1.1): max CoreDNS version to be used for upgrades v1.8.6